spec/requests/api/v1/users/games/request_spec.rb

```diff
@@ -3,6 +3,85 @@
 require 'rails_helper'
 
 RSpec.describe 'Api/V1/Users/Games/Request', type: :request do
+  describe 'GET /users/games' do
+    context 'when passed a user_id of a user with games' do
+      it 'returns a list of user games' do
+        user = create(:user)
+        game1 = create(:game)
+        game2 = create(:game)
+        game3 = create(:game)
+        create(:user_game, user: user, game: game1)
+        create(:user_game, user: user, game: game2)
+        create(:user_game, user: user, game: game3)
+
+        get api_v1_users_games_path, params: { user_id: user.id }
+
+        json_body = JSON.parse(response.body, symbolize_names: true)
+
+        expect(response.status).to eq(200)
+        expect(json_body).to be_a(Hash)
+        expect(json_body).to have_key(:data)
+        expect(json_body[:data]).to be_a(Array)
+        expect(json_body[:data].count).to eq(3)
+        expect(json_body[:data][0][:attributes]).to be_a(Hash)
+
+        json_body[:data].each do |item|
```

Comment:
We can put all of this in the response checkers doc. Doesn't have to be now though. We can do it during test cleanup.